Python Scraper FR still fails

cc @baryluk 

```
Traceback (most recent call last):
  File "././scrape_fr.sh", line 49, in <module>
    assert m, "Can't find table"
AssertionError: Can't find table
FR -                      -       - FAILED 2020-03-27T09:23:46+01:00 Missing date. Missing cases https://www.fr.ch/covid19/sante/covid-19/coronavirus-statistiques-evolution-de-la-situation-dans-le-canton
```

--- a/scrapers/scrape_fr.sh
+++ b/scrapers/scrape_fr.sh
@@ -66,7 +66,7 @@ d = d.replace('&nbsp;', '')
 """
 
 # For some magical reasons, it works without re.MULTILINE | re.DOTALL.
-r = re.search(r'<tr>\s*<td.*?>(\d\d\.\d\d\.\d\d)</td>\s*<td>([0-9]+| *)</td>\s*<td.*?>([0-9]+| *)</td>\s*<td.*>([0-9]+| *)</td>', d, flags=re.I)
+r = re.search(r'<tr>\s*<td.*?>(\d\d\.\d\d\.\d\d)</td>\s*<td.*?>([0-9]+| *)</td>\s*<td.*?>([0-9]+| *)</td>\s*<td.*?>([0-9]+| *)</td>', d, flags=re.I)
 assert r, "First row missmatch"
 
 print("Date and time:", r[1])

--- a/scrapers/scrape_fr.sh
+++ b/scrapers/scrape_fr.sh
@@ -45,12 +45,12 @@ sc.timestamp()
 </div>
 """
 
-m = re.search(r'<table class="table table-striped.*">(.*?)</table>', d, flags=re.MULTILINE | re.DOTALL)
+m = re.search(r'<table class="table.*?">(.*?)</table>', d, flags=re.MULTILINE | re.DOTALL)
 assert m, "Can't find table"
 
 d = m[1]
 
-header = sc.find("<tr>\s*(<th><strong>Date</strong></th>\s*<th><strong>Personnes hospitalisées</strong></th>\s*<th><strong>Total Décès</strong></th>\s*<th><strong>Total cas avérés</strong></th>)\s*</tr>", d)
+header = sc.find("<tr>\s*(<t[dh]><strong>Date</strong></t[dh]>\s*<t[dh]><strong>Personnes hospitalisées</strong></t[dh]>\s*<t[dh]><strong>Total Décès</strong></t[dh]>\s*<t[dh]><strong>Total cas avérés</strong></t[dh]>)\s*</tr>", d)
 assert header, "Header not matched"
 
 d = d.replace('&nbsp;', '')
@@ -66,7 +66,7 @@ d = d.replace('&nbsp;', '')
 """
 
 # For some magical reasons, it works without re.MULTILINE | re.DOTALL.
-r = re.search(r'<tr>\s*<td.*>(\d\d\.\d\d\.\d\d)</td>\s*<td>([0-9]+| *)</td>\s*<td.*>([0-9]+| *)</td>\s*<td.*>([0-9]+| *)</td>', d, flags=re.I)
+r = re.search(r'<tr>\s*<td.*?>(\d\d\.\d\d\.\d\d)</td>\s*<td>([0-9]+| *)</td>\s*<td.*?>([0-9]+| *)</td>\s*<td.*>([0-9]+| *)</td>', d, flags=re.I)
 assert r, "First row missmatch"
 
 print("Date and time:", r[1])

--- a/scrapers/scrape_fr.sh
+++ b/scrapers/scrape_fr.sh
@@ -45,12 +45,12 @@ sc.timestamp()
 </div>
 """
 
-m = re.search(r'<table class="table table-condensed table-hover">(.*?)</table>', d, flags=re.MULTILINE | re.DOTALL)
+m = re.search(r'<table class="table table-striped.*">(.*?)</table>', d, flags=re.MULTILINE | re.DOTALL)
 assert m, "Can't find table"
 
 d = m[1]
 
-header = sc.find("<tr>\s*(<td><strong>Date</strong></td>\s*<td><strong>Personnes hospitalisées</strong></td>\s*<td><strong>Total Décès</strong></td>\s*<td><strong>Total cas avérés</strong></td>)\s*</tr>", d)
+header = sc.find("<tr>\s*(<th><strong>Date</strong></th>\s*<th><strong>Personnes hospitalisées</strong></th>\s*<th><strong>Total Décès</strong></th>\s*<th><strong>Total cas avérés</strong></th>)\s*</tr>", d)
 assert header, "Header not matched"
 
 d = d.replace('&nbsp;', '')
@@ -66,7 +66,7 @@ d = d.replace('&nbsp;', '')
 """
 
 # For some magical reasons, it works without re.MULTILINE | re.DOTALL.
-r = re.search(r'<tr>\s*<td>(\d\d\.\d\d\.\d\d)</td>\s*<td>([0-9]+| *)</td>\s*<td>([0-9]+| *)</td>\s*<td>([0-9]+| *)</td>', d, flags=re.I)
+r = re.search(r'<tr>\s*<td.*>(\d\d\.\d\d\.\d\d)</td>\s*<td>([0-9]+| *)</td>\s*<td.*>([0-9]+| *)</td>\s*<td.*>([0-9]+| *)</td>', d, flags=re.I)
 assert r, "First row missmatch"
 
 print("Date and time:", r[1])

